src/Time/StepChoosers/StepChooser.hpp

```diff
@@ -18,84 +19,76 @@ namespace Parallel {
 template <typename Metavariables>
 class GlobalCache;
 }  // namespace Parallel
-template <typename StepChooserRegistrars>
-class StepChooser;
 // IWYU pragma: no_forward_declare db::DataBox
-template <typename StepChooserRegistrars>
-class StepChooser;
 /// \endcond
 
+/// The intended use for a step chooser.  This is used to control the
+/// classes via factories.
+namespace StepChooserUse {
+struct Slab;
+struct LtsStep;
+}  // namespace StepChooserUse
+
+/// \ingroup TimeSteppersGroup
+///
+/// StepChoosers suggest upper bounds on step sizes.  Concrete
```

Comment:
Make this \brief?

Reply:
By "this", do you mean the first sentence or the paragraph?

Reply:
Either works 🤷‍♂️